### ISSUE TITLE
Workspaces cannot be renamed

### ISSUE THREAD

**Issue by marktani** [prismagraphql/prisma-cloud-feedback#119]
Clicking "Save Changes" does nothing.

![image](https://user-images.githubusercontent.com/1780597/38077521-7e338f0e-3339-11e8-8187-1ed00fc10b00.png)

**Comment by Maxhodges** [comment]
seems you should just fix the UI. The UI appears to give you the ability to edit an existing workspace name. since it's can't be done, just show the workspace text as uneditable instead of presenting a UI with editable textfields

same with delete workspace. if it's the last workspace, maybe don't render a delete button?

**Comment by jonhorton** [comment]
Agreed with @Maxhodges. Why even show the form and a save changes button if it's not possible?

**Comment by timsuchanek** [comment]
Fixed in the latest version. Please let us know if you still have that issue, then we reopen it!

**Comment by marktani** [comment]
We reverted this feature due to an internal problem with changing workspace slugs.

Workspace names can still be adjusted.

**Comment by cmoses8626** [comment]
Any update on this? It'd be nice to be able to rename the slug. The tip above the form field that "the workspace slug can not be changed" is helpful, but admittedly confusing at first to read that and see what seems to be an editable form field before attempting to actually edit it. Agree a simple UI fix to not use a form field for the slug would be a step in the right direction.
